app/helpers/marks_graders_helper.rb

```diff
@@ -1,4 +1,35 @@
 module MarksGradersHelper
+  def get_marks_graders_table_info(grade_entry_form)
+    graders = Ta.all
+
+    graders.map do |grader|
+      s = grader.attributes
+      s[:num_students] = grader.get_membership_count_by_grade_entry_form(grade_entry_form)
+      s
+    end
+  end
+
+  def get_marks_graders_student_table_info(students, grade_entry_form)
```

Comment:
Assignment Branch Condition size for get_marks_graders_student_table_info is too high. [15.3/15]